ui/src/App.js

```diff
@@ -107,15 +107,15 @@ class App extends Component {
     }
```

Comment:
Github won't let me leave a comment on line 34. "search" isn't used elsewhere, so can line 34 change to
```
// Map from facet name to list of facet values
this.filterMap = new Map();
```

Reply:
Done